contracts/container/config.yml

```diff
@@ -28,3 +28,7 @@ events:
     parameters:
       - name: epoch
         type: Integer
+  - name: ContainerUpdate
+    parameters:
+      - name: containerID
```

Comment:
Names _are_ usually capitalized for event parameters.

Reply:
as usual, i prefer to change things in sync. can you, please, review this repo and create issues about things that are made weird for contracts? i cannot imagine how i should figure it out by myself. as you see, it is "epoch" few lines above

Reply:
OK, I'm wrong here, confused parameter name with event name. See https://github.com/neo-project/proposals/blob/master/nep-17.mediawiki for some reference.